ephios/core/services/notifications/types.py

```diff
@@ -239,16 +239,20 @@ def send(cls, participation: AbstractParticipation):
         )
         notifications = []
         for user in responsible_users:
-            notifications.append(
-                Notification(
-                    slug=cls.slug,
-                    user=user,
-                    data=dict(
-                        participation_id=participation.id,
-                        disposition_url=disposition_url,
-                    ),
+            if (
+                not participation.get_real_instance_class() == LocalParticipation
```

Comment:
don't they recommend `is_instance` for this?

Reply:
i remember that we already had this discussion once. We have another spot in the code where we use this and I couldn't find smth in the docs